src/main/java/seedu/storemando/model/item/Item.java

```diff
@@ -28,7 +28,7 @@
      * Every field must be present and not null.
      */
     public Item(ItemName name, Quantity quantity, ExpiryDate expiryDate, Location location, Set<Tag> tags) {
-        requireAllNonNull(name, quantity, expiryDate, location, tags);
+        requireAllNonNull(name, quantity, location, tags);
```

Comment:
Should this nonNull check for expiryDate be removed?

Reply:
I will add it back. expiryDate will not be null anymore.